js/msp.js

```diff
@@ -407,9 +409,9 @@ var MSP = {
                 }
                 if (semver.gte(CONFIG.apiVersion, "1.10.0")) {
                     RC_tuning.RC_YAW_EXPO = parseFloat((data.getUint8(offset++) / 100).toFixed(2));
-                    if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
+                    if (semver.gte(CONFIG.flightControllerVersion, "2.9.1")) {
```

Comment:
Lets do later cleanup for this. For 2.9.1 this is needed as it uses same API as 2.9.0.

We need to get rid of API stuff I think